docs/hooks/lua.md

```diff
@@ -317,6 +317,63 @@ The `value` string should be in [ISO8601](https://en.wikipedia.org/wiki/ISO_8601
 
 Returns a new 128-bit [RFC 4122 UUID](https://www.rfc-editor.org/rfc/rfc4122){: target="_blank" } in string representation.
 
+### `net/http` (optional)
+
+Provides a `request` function that performs an HTTP request.
+For security reasons, this package is not available by default as it enables http requests to be sent out from the lakeFS instance network. The feature should be enabled under `actions.lua.net_http_enabled` [configuration](../reference/configuration.md).
+Request will time out after 30 seconds.
+
+```lua
+http.request(url [, body])
+http.request{
+  url = string,
+  [method = string,]
+  [headers = header-table,]
+  [body = string,]
+}
+```
+
+Returns a code (number), body (string), headers (table) and status (string).
+
+ - code - status code number
+ - body - string with the response body
+ - headers - table with the response request headers (key/value or table of values)
+ - status - status code text
+
+The first form of the call will perform GET request or POST in case body passed to the request.
+The second form allows to customize the specific method and the request headers by passing a table with request information.
```

Comment:
Is it supported now? It's unclear from this sentence and the examples are lacking.

Reply:
Small update the the doc - hope it is clear now